### PR TITLE
Minor corrections to databricks docs

### PR DESCRIPTION
**Pull request by talSofer**

### CODE REVIEW

File: docs/integrations/databricks.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ In this document, we will cover the various Databricks products and how they int
 with lakeFS.
 
 <p class="center">
-    <img src="{{ site.baseurl }}/assets/img/Databricks-arch.png"/>
+    <img src="{{ site.baseurl }}/assets/img/databricks-arch.png"/>
```
Comment (itaiad200) [inline docs/integrations/databricks.md:20]: Already fixed

File: docs/integrations/databricks.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ the Unity Catalog metadata.
 
 lakeFS support for Unity Catalog differs between lakeFS OSS and lakeFS Enterprise & Cloud.
 
-### Catalog Exports <span class="badge">Open-Source</span>
+### Catalog Exports <span class="badge">lakeFS</span>
```
Comment (itaiad200) [inline docs/integrations/databricks.md:83]: I think this is `lakeFS Core` now? @ozkatz

File: docs/integrations/databricks.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ a way to export changes from lakeFS to Unity Catalog.
 
 For the full guide on how to use Catalog Exports with Unity Catalog, see the [documentation](./unity-catalog.md).
 
-### lakeFS for Databricks <span class="badge">Cloud</span> <span class="badge">Enterprise</span>
+### lakeFS for Databricks <span class="badge">Enterprise</span>
```
Comment (itaiad200) [inline docs/integrations/databricks.md:97]: Should be Enterprise and Managed Enterprise? @ozkatz

**Review by itaiad200 — CHANGES_REQUESTED** [review]

File: docs/integrations/databricks.md (hunk 2 above)
Comment (talSofer) [reply docs/integrations/databricks.md:83]: Decided on lakeFS

File: docs/integrations/databricks.md (hunk 3 above)
Comment (talSofer) [reply docs/integrations/databricks.md:97]: yes, but I think that it may be too wordy to mention both here. WDYT?

**Review by itaiad200 — APPROVED** [review]
LGTM
